main.rb

```diff
@@ -1,10 +1,13 @@
 require './app'
 require './create_service'
 require './list_service'
+require './storage'
 
 @app = App.new
+@storage = Storage.new
 
 def main
+  @storage.load_data(@app)
```

Comment:
- [ ] Kindly verify that all the required data is being loaded on the startup of your app. Right now that does not happen. Kindly fix this and make sure it works as expected 💯